extensions/amp-consent/0.1/consent-config.js

```diff
@@ -320,15 +322,21 @@ export class ConsentConfig {
  * Expand consent endpoint url
  * @param {!Element|!ShadowRoot} element
  * @param {string} url
+ * @param {Object<string, *>=} opt_vars
  * @return {!Promise<string>}
  */
-export function expandConsentEndpointUrl(element, url) {
+export function expandConsentEndpointUrl(element, url, opt_vars) {
```

Comment:
Rename `opt_vars` to `vars` and assign it a default value of `{}`.

Reply:
Sounds good.